<?xml version="1.0"?>
<launch>
  <arg name="config_path" default="$(find config)/vehicle" />
  <arg name="sensor_meta"
    default="$(arg config_path)/sensor/calibrated_sensor.pb.txt" />

  <arg name="obs_enable_hdmap_input" default="true" />
  <arg name= "fusion_config_path" default="$(find perception_fusion2)/config/" />

  <node name="perception_fusion2" ns="/perception/fusion"
    pkg="perception_fusion2" type="perception_fusion2_node" output="screen"
    args="--logtostderr=0 --stderrthreshold=0 --v=1
          --sensor_meta_path=$(arg sensor_meta) 
          --multi_sensor_fusion_conf_path=$(arg config_path)/perception/fusion/multi_sensor_fusion2.pb.txt
          --work_root=$(arg fusion_config_path)" >
    <param name="use_filter" type="bool" value="true"/>
  </node>
</launch>

